src/plugins/intel_gpu/include/intel_gpu/runtime/debug_configuration.hpp

```diff
@@ -25,6 +25,8 @@ class debug_configuration {
     debug_configuration();
 
 public:
+    static std::string get_device_id();
+    static const char *_device_id;
```

Comment:
Please add a comment

Reply:
Updated to use one integer device_id variable. Therefore I think it's clear enough to understand it without comment now.